test/Collation.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import { render } from "@testing-library/react";
+import { it, describe, expect } from "vitest";
+import Message from "../src/messages/Message";
+
+describe("Collation", () => {
+	it("collates if timestamp is in limit", () => {
+		const message1 = { text: "Hello", source: "bot", timestamp: Date.now() - 1000 * 50 };
```

Comment:
here you can simple infer IMessage like in the other specs:
`const message1 = { text: "Hello", source: "bot", timestamp: Date.now() - 1000 * 50 } as unknown as IMessage;`

Reply:
fixed